Sources/StreamChat/Database/DTOs/ChannelDTO_Tests.swift

```diff
@@ -7,6 +7,8 @@ import XCTest
 
 class ChannelDTO_Tests: XCTestCase {
     var database: DatabaseContainer!
+
+    static let channelCreatedDate = "2019-12-12T15:33:46.488935Z".toDate()
```

Comment:
To make this more fun, you can make this `.unique`, too. It will be unique for the duration of the tests, but it will change with every test suite run to provide additional variability in the tests.